### PR TITLE
Add possibility in example to ignore cert errors and fix the verify_ssl assignment in code

### PR DESCRIPTION
**Pull request by bunnis**
This addresses #298, adding it explicitly to the example and making it easier for new users to toggle it.
Additionally, the TCPConnector assignment was broken, and this was fixed too.

### CODE REVIEW

File: sagemcom_api/client.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ def __init__(
             else ClientSession(
                 headers={"User-Agent": f"{DEFAULT_USER_AGENT}"},
                 timeout=ClientTimeout(DEFAULT_TIMEOUT),
-                connector=TCPConnector(verify_ssl=verify_ssl if verify_ssl else True),
+                connector=TCPConnector(verify_ssl if verify_ssl is not None else True),
```
Comment (iMicknl) [inline sagemcom_api/client.py:108]: Can you explain why this change is required? Does verify_ssl = False not work?

Comment (bunnis) [reply sagemcom_api/client.py:108]: the logic here says something like, `variable=valueA if condition, else valueB`. If `verify_ssl=False`, then it reads, `verify_ssl=False if **False**, else True.` Since False always evaluates to False, we are never able to set the variable to False.

My logic fixes the evaluation, see below example in python interpreter:
`
>>> verify_ssl=False
>>> verify_ssl if verify_ssl else True
True
>>> verify_ssl=True
>>> verify_ssl if verify_ssl else True
True


>>> verify_ssl if verify_ssl is not None else True
True
>>> verify_ssl=False
>>> verify_ssl if verify_ssl is not None else True
False
`

Comment (iMicknl) [reply sagemcom_api/client.py:108]: Good catch! Makes sense.

**Review by iMicknl — CHANGES_REQUESTED** [review]
Can you run pre-commit on both your PRs? Both PRs are not passing CI/CD due to style issues.

**Comment by bunnis** [comment]
Could you let me know how I do that?

**Comment by iMicknl** [comment]
> Could you let me know how I do that?

If you use the devcontainer, it should be there by default where you commit. Otherwise you can run `pre-commit run --all-files`. I fixed them for you :).

**Review by iMicknl — APPROVED** [review]

**Comment by bunnis** [comment]
Great thank you, sorry for that, I don't really know much about these tools
